src/control/SConscript

```diff
@@ -203,6 +207,15 @@ def scons():
 
     denv.AppendENVPath("CGO_CFLAGS", denv.subst("$_CPPINCFLAGS"), sep=" ")
 
+    dbenv = denv.Clone()
+    dblibs = dbenv.subst("-L$BUILD_DIR/src/gurt "
+                         "-L$BUILD_DIR/src/cart "
+                         "-L$BUILD_DIR/src/client/dfs "
+                         "-L$BUILD_DIR/src/utils $_RPATH")
+    dbenv.AppendENVPath("CGO_LDFLAGS", dblibs, sep=" ")
+    daosbin = install_go_bin(dbenv, gosrc, [daos_hdlrs_lib], "daos", "daos")
```

Comment:
(pylint-undefined-variable) Undefined variable 'daos_hdlrs_lib'

Reply:
This looks to be a real issue, and the build isn't working with the new client build code from @jolivier23.

Build logs are at https://github.com/daos-stack/daos/runs/2832318905?check_suite_focus=true

Reply:
Sigh, yes. Fixing it.

Reply:
The only reason this isn't a build failure is that daos under github-actions it's 100% stable yet so this step is not marked as required, but we should be enabling it shortly.